django_stomp/helpers.py

```diff
@@ -115,3 +118,31 @@ def is_heartbeat_enabled(outgoing_heartbeat: int, incoming_heartbeat: int):
     More on: https://stomp.github.io/stomp-specification-1.1.html#Heart-beating
     """
     return outgoing_heartbeat > 0 and incoming_heartbeat > 0
+
+
+def set_ssl_connection(conn: StompConnection11) -> StompConnection11:
+    """Sets the SSL connection params given some values and return it"""
+
+    from django_stomp import settings
+
+    host_and_ports = settings.STOMP_HOST_AND_PORTS
+    if not host_and_ports:
```

Comment:
I didn't understand the need for this environment variable, by default it joins the values of `STOMP_SERVER_HOST` and `STOMP_SERVER_PORT`, but in what situation would I overwrite them to use only `STOMP_HOST_AND_PORTS`.

Another point is when you create the environment variable with the values `[("host", 61610)]`, when you get that value with the `os.getenv` function, no treatment is needed to transform that value into a list with a tuple?

Reply:
You have a point here, I think that I'll leave that this `STOMP_HOST_AND_PORTS = [(STOMP_SERVER_HOST, STOMP_SERVER_PORT)]`, what do you think? Because there is really no need to overwrite this variable, since the work is being done here in the lib. Thx!